disabled_apps/clock/manifest.webmanifest

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "Clock",
+  "description": "Gaia Clock",
+  "lang": "en",
+  "short_name": "Clock",
+  "scope": "./",
```

Comment:
The scope is resolved against the manifest url, so this should work, since our manifest and our index.html sit in the same directory. I think it's more convenient to keep the scope url relative.

https://www.w3.org/TR/appmanifest/#scope-member